src/runtime_src/core/common/shim/buffer_handle.h

```diff
@@ -39,6 +40,9 @@ class buffer_handle
   };
 
 public:
+  XCL_DRIVER_DLLESPEC
```

Comment:
As buffer_handle class is inherited by shim buffer_object classes and when shim objects are constructed constructor of buffer_handle is called and if we don't export it throws linker error in windows flow. That said this goes away as I will move logging to bo_impl class instead of buffer_handle